### PR TITLE
[Insert] Return more info of insert operation

### PR DESCRIPTION
**Pull request by morningman**
Standardize the return results of INSERT operations,
which is convenient for users to use and locate problems.

More details can be found in insert-into-manual.md

Also add a new stmt: SHOW TRANSACTION that user can see
the transaction status of the specified transaction id.

ISSUE: #2717

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        }
+
+        if (whereClause == null) {
+            throw new AnalysisException("Missing transaction id");
+        }
+
+        analyzeSubPredicate(whereClause);
+    }
+
+    private void analyzeSubPredicate(Expr subExpr) throws AnalysisException {
```
Comment (EmmyMiao87) [inline fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java]: analyzeWhereClause

File: fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        if (subExpr == null) {
+            return;
+        }
+
+        boolean valid = true;
+        CHECK: {
+            if (subExpr instanceof BinaryPredicate) {
```
Comment (EmmyMiao87) [inline fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java]: if the stmt is where 1=id?

File: fe/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 3]
```diff
@@ -692,37 +697,36 @@ private void handleInsertStmt() throws Exception {
...
+                    throwable == null ? "" : throwable.getMessage(),
+                    coord.getTrackingUrl());
+        } catch (MetaNotFoundException e) {
+            LOG.warn("Record info of insert load with error {}", e.getMessage(), e);
+            errMsg = "Record info of insert load with error " + e.getMessage();
+        }
+
+        // {'label':'my_label1', 'status':'visible', 'txnId':'123'}
+        // {'label':'my_label1', 'status':'visible', 'txnId':'123' 'err':'error messages'}
+        String info = "{'label':'" + label + "', 'status':'" + txnStatus.name() + "', 'txnId':'"
```
Comment (EmmyMiao87) [inline fe/src/main/java/org/apache/doris/qe/StmtExecutor.java]: StringBuilder.append()

File: fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java [hunk 4]
```diff
@@ -1220,6 +1214,63 @@ private void updateDBRunningTxnNum(TransactionStatus preStatus, TransactionState
...
+            if (db == null) {
+                throw new AnalysisException("Database[" + dbId + "] does not exist");
+            }
+            
+            TransactionState txnState = idToTransactionState.get(txnId);
+            if (txnState == null) {
+                throw new AnalysisException("transaction with id " + txnId + " does not exist");
+            }
+            
+            if (ConnectContext.get() != null) {
```
Comment (EmmyMiao87) [inline fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java]: precondition?

File: fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java (hunk 2 above)
Comment (morningman) [reply fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java]: Not support it~

File: fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java (hunk 1 above)
Comment (morningman) [reply fe/src/main/java/org/apache/doris/analysis/ShowTransactionStmt.java]: ok

File: fe/src/main/java/org/apache/doris/qe/StmtExecutor.java (hunk 3 above)
Comment (morningman) [reply fe/src/main/java/org/apache/doris/qe/StmtExecutor.java]: ok

File: fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java (hunk 4 above)
Comment (morningman) [reply fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java]: This method may be called internal, not in user connection thread. So I just add a judgement here.

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**